README-pt-br.md

```diff
@@ -0,0 +1,1730 @@
+# What the f\*ck JavaScript?
+
+[![WTFPL 2.0][license-image]][license-url]
+[![NPM version][npm-image]][npm-url]
+
+> Uma lista de exemplos engraçados e truques com JavaScript
+
+JavaScript é uma excelente linguagem. Ela tem uma sintaxe simples, um ecossistema grande e, o mais importante, uma grande comunidade.
+
+Ao mesmo tempo, todos nós sabemos que o JavaScript é uma linguagem engraçada com várias partes complicadas. Algumas delas porem rapidamente transformar seu trabalho em um inferno, e outras podemo nos fazer gargalhar.
+
+A ideia original para o WTFJS é do [Brian Leroux](https://twitter.com/brianleroux). Essa lista é inspirada por sua talk [**“WTFJS”** no dotJS 2012](https://www.youtube.com/watch?v=et8xNAc2ic8):
+
+[![dotJS 2012 - Brian Leroux - WTFJS](https://img.youtube.com/vi/et8xNAc2ic8/0.jpg)](https://www.youtube.com/watch?v=et8xNAc2ic8)
+
+# Node Packaged Manuscript
+
+Você pode instalar esse manual usando o `npm`. É só rodar o comando:
+
+```
+$ npm install -g wtfjs
+```
+
+Você poderá rodar `wtfjs` na sua linha de comando. Esse comando vai abrir o manual na sua `$PAGER` selecionada. Ou, você pode continuar lendo aqui mesmo.
```

Comment:
"selecionada ou você pode continuar lendo aqui mesmo"

Reply:
Hi @cassiodias, thanks for the review.
I agree, the changes make for a better Portuguese text. It's my first translation, I didn't know how much I could change from the original style in order to make it feel more natural in Portuguese, but definitely it sounds better the way you propose. :-)